addon/ng2/blueprints/component/files/__path__/__name__.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component} from 'angular2/core';
+
+@Component({
+  moduleId: __moduleName,
+  selector: '<%= dasherizedModuleName %>',
+  templateUrl: '<%= dasherizedModuleName %>.component.html',
+  styleUrls: ['<%= dasherizedModuleName %>.component.<%= styleExt %>']
+})
+export class <%= classifiedModuleName %>Component {
```

Comment:
Yes, I prefer it to be there as well. Seems like a great suggestion for making your code more testable.